client/web/src/views/ChartViewContent/charts/bar/BarChart.tsx

```diff
@@ -149,6 +149,7 @@ export function BarChart<Datum extends object>(props: BarChartProps<Datum>): Rea
                                 <MaybeLink
                                     key={`bar-${index}`}
                                     to={linkURLs?.[index]}
+                                    target="_blank"
```

Comment:
See https://love2dev.com/blog/rel-noopener/
```suggestion
                                    target="_blank"
                                    rel="noopener"
```